### PR TITLE
Change colorscheme of `pod outdated` from red-yellow-green to red-blue-green

### PR DESCRIPTION
**Pull request by iv-mexx**
This is an improvement for colorblind users who can not differentiate between green and yellow.
Refs #7372 

As discussed in #7372 there might be a more comprehensive solution in the future. 

🌈

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* [WIP] Change colorscheme of `pod outdated` from red-yellow-green to red-blue-green  
  [iv-mexx](https://github.com/iv-mexx)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
I am OK with this but it seems incomplete compared to all cocoapods commands running. Again if possible a more comprehensive way feels better.

I will let others weigh in a bit.

**Comment by amorde** [comment]
I think this is a good start because it is one of the few commands that use 3 separate colors in section of output. I am curious about red-green colorblindness is affecting by this as well

**Comment by dnkoutso** [comment]
Needs rebase

**Comment by iv-mexx** [comment]
@dnkoutso Alright, I have rebased onto current master

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   resources or embed frameworks unless `UNLOCALIZED_RESOURCES_FOLDER_PATH` 
   or `FRAMEWORKS_FOLDER_PATH` is set.  
   [Samuel Giddins](https://github.com/segiddins)
+* Change color scheme of `pod outdated` from red-yellow-green to red-blue-green to be more colorblind friendly
```
Comment (dnkoutso) [inline CHANGELOG.md]: nit: press enter

Comment (iv-mexx) [reply CHANGELOG.md]: 👍

**Comment by segiddins** [comment]
🚢 , thanks!

**Pull request merged**